src/index.css

```diff
@@ -22,3 +22,22 @@
     @apply text-[2.5rem];
   }
 }
+
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+
+@layer utilities {
+  .link-button-font {
+    @apply text-[3rem];
+  }
+  .link-button-height {
+    @apply h-[44px];
+  }
+  .link-button-width {
```

Comment:
Normally you should add width to button, because you have paddings for this, but okay.

Reply:
Removed height, and left only line-height to vertically align text in a Link.